monitoring/explores/database_sizes_mb.explore.lkml

```diff
@@ -0,0 +1,5 @@
+include: "../views//database_sizes_mb.view.lkml"
+
+ explore: database_sizes_mb {
+   view_name: database_sizes_mb
```

Comment:
Since we only use this explore for some internal analysis, we can set it to `hidden`. This means the explore won't be explicitly listed in the "Explore"s menu and reduces clutter. 

```suggestion
   view_name: database_sizes_mb
   hidden: yes
```